examples/fixed_editor.ml

```diff
@@ -275,6 +275,7 @@ let window1 () =
 
     
 let main () =
+  let _ = Main.init () in
```

Comment:
This `Main.init ()` is not needed. Examples are intended to be run with `lablgtk2`, which already calls it.
Alternatively, one can link `gtkInit.cmo`, or use the `auto-init` package.

Reply:
I see, thanks. WIthout it the program was segfaulting; I will move these changes to a different PR then.